src/frontend/src/optimizer/plan_node/logical_source.rs

```diff
@@ -46,6 +46,7 @@ use crate::utils::{ColIndexMapping, Condition, IndexRewriter};
 /// [`DataType::Timestamptz`]. For more details, please refer to
 /// [this rfc](https://github.com/risingwavelabs/rfcs/pull/20).
 pub const KAFKA_TIMESTAMP_COLUMN_NAME: &str = "_rw_kafka_timestamp";
+pub const KAFKA_KEY_COLUMN_NAME: &str = "_rw_kafka_key";
```

Comment:
other mq also have key part. why limit to kafka

Reply:
But should we channge the `_rw_kafka_timestamp` too? or the columns will be named with the different style.